### PR TITLE
Add navbar

### PR DESCRIPTION
**Pull request by shanialmog**
check it out

### CODE REVIEW

File: src/components/Login/index.js [hunk 1]
```diff
@@ -1,30 +1,59 @@
...
+    //         }
+    //         // console.log(response)
+    //         let textResponse = await (response.json())
+    //         // console.log("is logged in ", textResponse)
+    //         setLoggedIn(true)
+    //         setUserName(textResponse.user.username)
+    //         setFetchData(false)
+    //     }
+    //     isLoggedIn();
+    // }, [])
```
Comment (guyb7) [inline src/components/Login/index.js:44]: Don't commit commented out code

File: src/components/Login/index.js [hunk 2]
```diff
@@ -1,30 +1,59 @@
...
 
     const handleChange = (event) => {
         const { name, value } = event.target
-        if (name === 'username') {
-            setUserName(value)
+        if (name === 'userName') {
+            setTempUserName(value)
         } else { setPassword(value) }
+        console.log("tempUserName",tempUserName)
+        console.log("userName",userName)
```
Comment (guyb7) [inline src/components/Login/index.js:56]: Don't commit debugging code

File: src/components/NavBar/index.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+            // }
+            // console.log(response)
+            let textResponse = await (response.json())
+            console.log("is logged in ", textResponse)
+            if (textResponse.loggedIn) {
+                setIsLoggedIn(true)
+                setUserName(textResponse.user.userName)
+            }
+            // setFetchData(false)
+        }
```
Comment (guyb7) [inline src/components/NavBar/index.js:42]: Why do you need to wrap this code in a function?

File: src/App.js [hunk 4]
```diff
@@ -2,28 +2,34 @@ import React from 'react'
...
 import Event from './components/Event'
 import Dashboard from './components/Dashboard'
 import NewEvent from './components/NewEvent'
 
+
+
 function App() {
   return (
-    <div>
+    <Provider value={10}>
```
Comment (guyb7) [inline src/App.js:18]:
```suggestion
    <Provider>
```

File: src/App.js [hunk 5]
```diff
@@ -2,28 +2,34 @@ import React from 'react'
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom'
 import CssBaseline from '@material-ui/core/CssBaseline'
 
+import Provider from './store/UserContext'
```
Comment (guyb7) [inline src/App.js:5]:
```suggestion
import UserContext from './store/UserContext'
```

File: src/components/NavBar/index.js (hunk 3 above)
Comment (shanialmog) [reply src/components/NavBar/index.js:42]: it has to be wrapped in a function to be async.

File: src/components/NavBar/index.js [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
```
Comment (guyb7) [inline src/components/NavBar/index.js:42]:
```suggestion
    useEffect(async () => {
        const response = await fetch('/users/status')
        const textResponse = await (response.json())
        if (textResponse.loggedIn) {
            setIsLoggedIn(true)
            setUserName(textResponse.user.userName)
        }
```

File: src/components/NavBar/index.js [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+
+
+const NavBar = () => {
+    const user = useContext(UserContext)
+    const [isLoggedIn, setIsLoggedIn] = user.isLoggedIn
+    const [userName, setUserName] = user.userName
+
+    
+    useEffect(() => {
+        console.log("context-user", isLoggedIn)
```
Comment (guyb7) [inline src/components/NavBar/index.js:20]: Remove

File: src/components/NavBar/index.js [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+        const checkLogin = async () => {
+            let response = await fetch('/users/status')
+            let textResponse = await (response.json())
+            console.log("is logged in ", textResponse)
```
Comment (guyb7) [inline src/components/NavBar/index.js:24]: Remove

File: src/components/NavBar/index.js [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+    const user = useContext(UserContext)
+    const [isLoggedIn, setIsLoggedIn] = user.isLoggedIn
+    const [userName, setUserName] = user.userName
+
+    
+    useEffect(() => {
+        console.log("context-user", isLoggedIn)
+        const checkLogin = async () => {
+            let response = await fetch('/users/status')
+            let textResponse = await (response.json())
```
Comment (guyb7) [inline src/components/NavBar/index.js:23]:
```suggestion
            const response = await fetch('/users/status')
            const textResponse = await (response.json())
```

**Pull request merged**